cvat-ui/src/components/create-task-page/advanced-configuration-form.tsx

```diff
@@ -276,15 +280,37 @@ class AdvancedConfigurationForm extends React.PureComponent<Props> {
         );
     }
 
+    private renderGitFormat(): JSX.Element {
+        const { dumpers } = getCVATStore().getState().formats.annotationFormats.dumpers;
```

Comment:
It's a bad practice to use state directly, better to put dumpers in `mapStateToProps` function. Also I have an error have due to dumpers always is undefined.